src/lib/session/session.ts

```diff
@@ -14,7 +15,7 @@ export async function sessionGetTenant(): Promise<string | undefined> {
 export async function sessionMustGetTenant(): Promise<string> {
   const tenant = await sessionGetTenant();
   if (!tenant) {
-    throw new Error("Tenant not found in session");
```

Comment:
This should not silently redirect. It should throw because it is a low level library. Page, middleware or something else should handle the error and redirect. A low level library like this should not make decision as to redirect to where. May be the page wants to redirect to `/x` if session is not there.